crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/model/AuthorizationResponse.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.laa.crime.meansassessment.model;
+
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AuthorizationResponse {
```

Comment:
There is already an AuthorizationResponseDTO class.

Reply:
Okay. removed the duplicated class.